flatbuffers/serial/rootvalue.fbs

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+include "collation.fbs";
+
+namespace serial;
+
+table RootValue {
+  feature_version:int64;
+
+  tables:[ubyte]; // Serialized AddressMap.
+
+  foreign_key_addr:[ubyte];
+
+  collation:Collation;
+  
+  // Schemas records the schemas in this database, which may be empty for a database with a single default schema
+  schemas:[DatabaseSchema];
+}
+
+table DatabaseSchema {
+  name:string;
+}
+
+// This should be equivalent to the identifier for Dolt's root value
+file_identifier "RTVL";
```

Comment:
Might want a different one here. I don't think there's any expectation that Dolt can read root values written by Doltgres

@reltuk